### PR TITLE
Adding interactive tutorials for VCDAT

### PR DESCRIPTION
**Pull request by downiec**
This update includes the use of my tutorial extension manager and some interactive tutorial steps designed for VCDAT. I also added an 'About' pop-up which can be accessed in the help menu. Also fixed issue #133 by closing varloader soon as load is clicked.

### CODE REVIEW

File: src/components/AboutVCDAT.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                Documentation
+              </a>
+            </Col>
+          </Row>
+        </ModalBody>
+        <ModalFooter>
+          <Row style={{ width: "100%", margin: 0 }}>
+            <Col>
+              <p
+                style={{
```
Comment (sterlingbaldwin) [inline src/components/AboutVCDAT.tsx]: I think any style that takes more then a single line should be declared at the top of the file.

File: src/index.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
 // Dependencies
+import * as React from "react";
```
Comment (sterlingbaldwin) [inline src/index.tsx]: Is there anything in this file that needs react?

**Review by sterlingbaldwin — APPROVED** [review]
99.9% looks good, I would approve as is. A couple tiny changes. Also Im not sure the Deepcopy method is strictly necessary, but its not terrible and doesnt hurt anything so no need to remove it for the time being.

File: src/index.tsx (hunk 2 above)
Comment (downiec) [reply src/index.tsx]: Good catch, I must have missed removing that.

File: src/components/AboutVCDAT.tsx (hunk 1 above)
Comment (downiec) [reply src/components/AboutVCDAT.tsx]: I'll fix that

**Pull request merged**